docs/nominatim.openapi.json

```
@@ -299,6 +299,171 @@
                     "Search"
                 ]
             }
+        },
+        "/lookup": {
+            "get": {
+                "summary": "Query the address and other details of one or multiple OSM objects like node, way or relation.",
+                "operationId": "123",
+                "description": "Returns the details of an OSM object.",
+                "parameters": [
+                    {
```

You totally missed the only required parameter `osm_ids` (see the [official docs](https://nominatim.org/release-docs/develop/api/Lookup/#endpoint))... 😄 

```suggestion
                "parameters": [
                    {
                        "schema": {
                            "type": "string",
                            "pattern": "^([RWN][0-9]+,?)+$"
                        },
                        "in": "query",
                        "name": "osm_ids",
                        "required": true,
                        "description": "Comma-separated list of OSM ids each prefixed with its type, one of node(N), way(W) or relation(R). Up to 50 ids can be queried at the same time."
                    },
                    {
```